scripts/container-load/container-manifest.yaml

```diff
@@ -0,0 +1,8 @@
+registries:
+  - public.ecr.aws/p0k3r4s4
+  - gcr.io/codepipes-staging
+
+images:
+  - name: sonarsource/sonar-scanner-cli
+    tag: latest
```

Comment:
This should be an array maybe?

Reply:
I thought it was - did I get the yaml syntax incorrect?
I was testing with something like this:
images:
  - name: image1
    tag: latest
  - name: image2
    tag: v1

Is that not correct?

Reply:
@dshelley66 I mean the tag should be array as well no? like this:
```
- name: golang
   tag: [1.19, 1.20, latest]
```
so that we can add multiple tags of one image as well

Reply:
ok I changed it to support this